config/prod/AMP-Mayo-Sinai-Bucket.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: amp-mayo-sinai
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "NDR"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: 'william.poehlman@sagebase.org'
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  SynapseUserName: 'wpoehlm'
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::563295687221:user/william.poehlman@sagebase.org'
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  EncryptBucket: 'true'
+  # (Optional) 'Enabled' to enable bucket versioning, default is 'Suspended'
+  BucketVersioning: 'Enabled'
```

Comment:
its fine if you want to enable versioning but just wanted to inform you that this may be costly.   

How am I charged for using Versioning? https://aws.amazon.com/s3/faqs/

Reply:
I will talk to Anna about our goals here and let you know when I'm sure about these decisions.  Thanks!